packages/deployment-cli/tsconfig.json

```diff
@@ -1,8 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "outDir": "./dist",
-    "sourceRoot": "src/"
-  },
-  "exclude": ["node_modules", "**/__tests__/*", "dist"]
+    "outDir": "dist",
+    "sourceRoot": "src"
+  }
```

Comment:
Should we still exclude `dist` here? Why remove? Would this break CI?

Reply:
It extends the root tsconfig which already has the exclude definitions. Tbh I'm not sure if why the `outDir` and `sourceRoot` are specified here and not in the root tsconfig as well. Unless that's becuase the paths need to be relative from this config file, I haven't really tried. The config is the same for all the apps

Reply:
> Unless that's becuase the paths need to be relative from this config file

I believe that is the case but also not sure